rdrf/rdrf/forms/progress/form_progress.py

```diff
@@ -16,7 +16,6 @@
 
 class ProgressType:
     DIAGNOSIS = "diagnosis"
-    GENETIC = "genetic"
```

Comment:
I'm not really familiar with the code in `form_progress.py`, but it might be worth removing `ProgressType` entirely and simplifying `FormProgress`

Reply:
@alintreznai Can you please add a JIRA task for this (removing ``ProgressType`` and simplifying ``form_progress``) and we tackle it later on?